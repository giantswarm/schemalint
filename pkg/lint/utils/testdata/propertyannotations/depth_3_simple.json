{
    "$defs": {
        "foo": {
            "properties": {
                "childProp": {
                    "type": "object",
                    "$ref": "#/$defs/bar",
                    "properties": {
                        "grandchildProp": {
                            "type": "string",
                            "title": "Grandchild Ref Prop"
                        }
                    }
                }
            },
            "type": "object"
        },
        "bar": {
            "properties": {
                "grandchildProp": {
                    "type": "string",
                    "title": "Grandchild Ref Ref Prop"
                }
            }
        }
    },
    "properties": {
        "rootProp": {
            "$ref": "#/$defs/foo",
            "properties": {
                "childProp": {
                    "type": "object",
                    "properties": {
                        "grandchildProp": {
                            "type": "string",
                            "title": "Grandchild Prop"
                        }
                    }
                }
            },
            "type": "object"
        }
    },
    "type": "object"
}
