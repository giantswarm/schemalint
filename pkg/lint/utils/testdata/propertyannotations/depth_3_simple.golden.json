{
  "/rootProp": {},
  "/rootProp/childProp": {},
  "/rootProp/childProp/grandchildProp": {
    "Title": {
      "Value": "Grandchild Prop",
      "ReferenceLevel": 0
    }
  }
}
